test/services/messengerSync/services/schoolSyncService.test.js

```diff
@@ -61,17 +59,41 @@ describe('messenger schoolSync Service', () => {
 			]);
 			const testingQueue = rabbitmqMock.queues.matrix_sync_unpopulated;
 			expect(testingQueue).to.not.be.undefined;
-			expect(testingQueue.length).to.equal(3); // 3 user creation events
+			expect(testingQueue.length).to.equal(4); // 3 user creation events + 1 school creation event
```

Comment:
```suggestion
			expect(testingQueue.length,' 3 user creation events + 1 school creation event').to.equal(4);
```
you may add comments directly if this fails